docs/configuration.md

```diff
@@ -38,6 +38,70 @@ config_3 = BdkConfigLoader.load_from_symphony_dir("config.yaml")  # 3
 ```
 1. Load configuration from a file
 2. Load configuration from a string object
-3. Load configuration from the Symphony directory. The Symphony directory is located under `.symphony` folder in your home directory 
-    . It can be useful when you don't want to share your own Symphony credentials within your project codebase
+3. Load configuration from the Symphony directory. The Symphony directory is located under `.symphony` folder in your
+   home directory. It can be useful when you don't want to share your own Symphony credentials within your project
+   codebase
 
+## Full configuration example
+```yaml
+scheme: https
+host: localhost.symphony.com
+port: 8443
+
+proxy:
+  host: proxy.symphony.com
+  port: 1234
+  username: proxyuser
+  password: proxypassword
+
+pod:
+  host: dev.symphony.com
+  port: 443
+
+agent:
+   host: dev-agent.symphony.com
+   port: 5678
+
+keyManager:
+  host: dev-key.symphony.com
+  port: 8444
+
+sessionAuth:
+  host: dev-session.symphony.com
+  port: 8444
+
+ssl:
+  trustStore:
+    path: /path/to/truststore.pem
```

Comment:
```yaml
ssl:
  trustStore:
    path: /path/to/truststore.pem
    password: changeit
```
From https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/configuration.md#full-configuration-example

Reply:
Indeed but I don't see how a password would be used.

Reply:
ah you are right, in Java we use a keystore and here a PEM file!

Reply:
Let's perhaps add this link to the documentation (in a comment next to the full example)? saying we support pem (cafile) parameter?

Reply:
Done below.